core/services/ocr2/plugins/ocr2keeper/integration_21_test.go

```diff
@@ -229,8 +226,6 @@ func TestIntegration_KeeperPluginLogUpkeep(t *testing.T) {
 	g.Eventually(listener, testutils.WaitTimeout(t), cltest.DBPollingInterval).Should(gomega.BeTrue())
 	done()
 
-	runs := checkPipelineRuns(t, nodes, 1)
```

Comment:
Have we explored any alternate assertions for non pipeline transmitter? Or do we feel they are covered by existing assertions

Reply:
I wasn't able to see how the other services assert their transmits, so I just assumed the existing assertions would be enough, but I can look into this further

Reply:
Verifying the upkeeps are actually performed i think should give similar coverage but I'm not sure that's done everywhere (e.g. the other place where this is removed)